src/test/java/org/icatproject/integration/TestRS.java

```diff
@@ -500,7 +500,7 @@ private JsonArray searchDatafiles(Session session, String user, String text, Dat
 	@Test
 	public void testVersion() throws Exception {
 		ICAT icat = new ICAT(System.getProperty("serverUrl"));
-		assertTrue(icat.getVersion().startsWith("4."));
+		assertTrue(icat.getVersion().startsWith("5."));
```

Comment:
This comment is not about this line but more about whether we should be adding some tests that create and use the new entities and relationships created? 

I'm new to looking at the icat.server code but I can see that the current level of testing done is not ideal and not all existing entities appear to be tested, so I was wondering what other people think about this?

Reply:
That is a general problem in `ìcat.server`: we do not have a complete test suite covering all features and the full schema.  Adding that would be a new project.  It would require a systematic approach and the design of a comprehensive test strategy.  I don't think this can be fixed in a PR considering only a subset of the schema.

Reply:
I agree with your comments. This is a wider problem and not something that we are going to resolve here, or even improve the situation much by just adding tests for these new additions.